torch/distributed/distributed_c10d.py

```diff
@@ -231,7 +232,7 @@ def supports_complex(reduceOp: ReduceOp) -> bool:
     return reduceOp not in denyList
 
 
-class Backend(str):
+class Backend(Enum, str):
```

Comment:
If this enum like, it should inherit from Enum. In the future, there will be a StrEnum class which perfectly fits this role which pyupgrade will handle.